src/Components/Facility/models.tsx

```diff
@@ -218,6 +218,27 @@ export interface CurrentBed {
   meta: Record<string, any>;
 }
 
+export interface consultationBedRequest {
+  bed?: string;
+  consultation: string;
+  limit?: number;
+  offset?: number;
+}
+
+export interface consultationBedResponse {
+  count: number;
+  next: string;
+  previous: string;
+  results: CurrentBed[];
+  is_occuiped?: boolean;
+  created_date?: string;
+  modified_date?: string;
+  name?: string;
+  description?: string;
+  meta?: Record<string, any>;
+  assets_objects?: AssetData[];
+}
```

Comment:
1. Interfaces/Types should be in PascalCase
2. Use PaginatedResponse type wrapper for paginated responses instead of manually defining count, next, previous.